examples/mobile/UIComponents/webclip/ads-card-3/css/style.css

```diff
@@ -0,0 +1,3 @@
+.app-banner-1 {
```

Comment:
I just have a question.
If the official GUI guide is released, the "app-banner-1" will be included in the tau css library, right?

Reply:
No. This class is only for this card and reference only to this specific image. Just this photo looks better (in my opinion) when is little higher positioned. The developer can adjust some elements according to him needs, I think.

Reply:
Ok. I see. Thanks.